### ISSUE TITLE
Create Card and Buttons for Participant Dashboard

### ISSUE THREAD

**Issue by b-f-chan** [OHCRN/platform#153]
Create the card with Complete Consents and Download Consents buttons for the Participant Dashboard

**Expected Outcomes:**

These cards exist:

- [x] Disabled state (pending email verification)
- [x] Enabled but consents not started/complete
- [x] Enabled and consents are completed

Also:

- [x] Hook up the translation

**Comment by patrickdsantos** [comment]
Wonderful work @alyssadsouza !!! The background image looks fantastic even on my fullscreen monitor!

Some feedback below:
1. For the white card on the homepage, the padding is equal but visually the bottom seems a bit unbalanced. Can we increase the padding at the bottom to 30px?
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN29DQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2f4f1b12be70069200ce4bec8c15dfa49881a9cd/Screenshot%202023-11-23%20at%2010.33.09%20AM.png)
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBN3NDQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--e9de669298cc0b4bea9a9df7a9dc6de1ef3f1e31/Screenshot%202023-11-23%20at%2010.32.52%20AM.png)

2. The overall styling of the cards is wonderful! As I was playing around with the responsive design, I noticed the card would get super stretched out on bigger screens. Is there a way to put a max width of 1200px to the entire card? Anything bigger would just increase the margins? Similar to what we did with the homepage styling. 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOVVDQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b276483ad852771554fa62eae3e2bb9b3bd74f89/Screenshot%202023-11-23%20at%2010.45.21%20AM.png)

3. Could we reduce the padding between the button and the bottom of the card? (If possible.) Let's go with reducing it by 50% and see how that looks like! 
[Private Zenhub Image](https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOW9DQWc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--7216055025fd8cdc213c9c9776b375f7b1e562ea/Screenshot%202023-11-23%20at%2010.52.05%20AM.png)

Thanks again Alyssa!!
